pkg/reconciler/revision/config/store_test.go

```diff
@@ -96,6 +106,7 @@ func TestStoreImmutableConfig(t *testing.T) {
 	store.OnConfigChanged(ConfigMapFromTestFile(t, pkgmetrics.ConfigMapName()))
 	store.OnConfigChanged(ConfigMapFromTestFile(t, logging.ConfigMapName()))
 	store.OnConfigChanged(ConfigMapFromTestFile(t, pkgtracing.ConfigName))
+	store.OnConfigChanged(ConfigMapFromTestFile(t, apisconfig.DefaultsConfigName))
```

Comment:
we should test for immutability of defaults config object too similar to whats done below:

https://github.com/knative/serving/blob/18933ceb4ce0be33383aedf677af68cf19f3031b/pkg/reconciler/revision/config/store_test.go#L113-L115